pkg/apis/navigator/v1alpha1/types.go

```diff
@@ -32,10 +32,26 @@ type CassandraClusterSpec struct {
 	CqlPort   int32                      `json:"cqlPort"`
 }
 
+// CassandraClusterPersistenceConfig contains persistent volume
+// configuration.
+type CassandraClusterPersistenceConfig struct {
```

Comment:
Can we rename `ElasticsearchClusterPersistenceConfig` to `PersistenceConfig` and re-use it between the two implementations?

This should also support the 'Enabled' field (like `ElasticsearchClusterPersistenceConfig` has) to allow users to deploy without PVs for testing purposes etc.